### PR TITLE
allows the use of stacked bars in multiChart

### PR DESCRIPTION
**Pull request by montrezorro**

### CODE REVIEW

File: src/models/multiChart.js [hunk 1]
```diff
@@ -204,17 +204,29 @@ nv.models.multiChart = function() {
...
-            var extraValue1 = dataStack1.length ? dataStack1.map(function(a){return a.values}).reduce(function(a,b){
-                return a.map(function(aVal,i){return {x: aVal.x, y: aVal.y + b[i].y}})
-            }).concat([{x:0, y:0}]) : [];
-            var extraValue2 = dataStack2.length ? dataStack2.map(function(a){return a.values}).reduce(function(a,b){
-                return a.map(function(aVal,i){return {x: aVal.x, y: aVal.y + b[i].y}})
-            }).concat([{x:0, y:0}]) : [];
-
-            yScale1 .domain(yDomain1 || d3.extent(d3.merge(series1).concat(extraValue1), function(d) { return d.y } ))
+            var extraValue1BarStacked = [];
+	        if (bars1.stacked() && dataBars1.length) {
```
Comment (liquidpele) [inline src/models/multiChart.js]: you're mixing tabs and spaces here... please only use 4-space indention.

**Comment by liquidpele** [comment]
ping...

**Comment by montrezorro** [comment]
sorry, I forget the pull-request. I've correct the indention

File: src/models/multiChart.js [hunk 2]
```diff
@@ -204,17 +204,29 @@ nv.models.multiChart = function() {
...
+                if (bars1.stacked() && dataBars1.length) {
+                    var extraValue1BarStacked = dataBars1.filter(function(d){return !d.disabled}).map(function(a){return a.values});
+                    
+                    if (extraValue1BarStacked.length > 0)
+                        extraValue1BarStacked = extraValue1BarStacked.reduce(function(a,b){
+                            return a.map(function(aVal,i){return {x: aVal.x, y: aVal.y + b[i].y}})
+                        }).concat([{x:0, y:0}]);
+                }
+                var extraValue2BarStacked = [];
+            if (bars2.stacked() && dataBars2.length) {
```
Comment (liquidpele) [inline src/models/multiChart.js]: The spacing is still off...    make sure everything is indented correctly...

**Comment by liquidpele** [comment]
Ping.   Is this PR abandoned?   I'll have to close it if so...

**Comment by montrezorro** [comment]
I have replaced the tab with space. What's wrong now?

**Comment by liquidpele** [comment]
The start of the if statement isn't lined up with the previous statement, and there isn't a closing bracket there to denote a change in indention.

**Comment by liquidpele** [comment]
Looks good now, thanks for contributing!

**Pull request merged**